data/tlg0010/tlg027/__cts__.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
    <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0010.tlg027" xml:lang="grc" groupUrn="urn:cts:greekLit:tlg0010">
-      <ti:title xml:lang="grc">To Philip, I</ti:title>
+      <ti:title xml:lang="eng">To Philip, I</ti:title>
       <ti:edition urn="urn:cts:greekLit:tlg0010.tlg027.perseus-grc2" workUrn="urn:cts:greekLit:tlg0010.tlg027" xml:lang="grc">
-         <ti:label xml:lang="eng">To Philip, I</ti:label>
-         <ti:description xml:lang="eng">Isocrates. Isocrates with an English Translation in three volumes, by Larue Van Hook, Ph.D., LL.D. Cambridge, MA, Harvard University Press; London, William Heinemann Ltd. 1945-1968.</ti:description>
+         <ti:label xml:lang="grc">Φιλίππῳ</ti:label>
```

Comment:
I wasn't sure what to do about a Greek title for this work. Both the first letter and the second letter have the same Greek titles with no indication of numbers.

Reply:
Greek numbers are α′ β′ etc. So:
Φιλίππῳ α΄
Φιλίππῳ β΄
as here https://el.wikisource.org/wiki/Επιστολές_(Ισοκράτης)

Reply:
hi @lcerrato thanks!